x/auth/client/cli/tx_multisign.go

```diff
@@ -91,7 +93,7 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 			return err
 		}
 
-		k, err := getMultisigRecord(clientCtx, args[1])
+		k, err := clientCtx.Keyring.Key(name)
 		if err != nil {
 			return err
```

Comment:
We should wrap the error `return nil, errorsmod.Wrap(err, "error getting keybase multisig account")`

Reply:
Thanks for the reminder.